src/main/java/pl/printo3d/onedcutter/cutter1d/cutter/services/OrderService.java

```diff
@@ -25,8 +25,6 @@
   @Autowired
   private UserService userService;
 
-  public OrderModel orderList = new OrderModel();
```

Comment:
Nie jest Ci to pole potrzebne w tej klasie, możesz tworzyć lokalną zmienną wewnątrz metody